Osiris/SDK/Entity.h

```diff
@@ -235,7 +235,33 @@ class Entity {
         callVirtualMethod<void>(this, 345, std::ref(vec));
         return vec;
     }
+    
+    bool isInThrow() noexcept {
+        auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+        if (!localPlayer || localPlayer->isAlive())
```

Comment:
learn order of boolean operations, this will return false if the player is alive.
should be `if (!localPlayer || !localPlayer->isAlive())`

Reply:
grenades won't work properly

Reply:
> if (!localPlayer || !localPlayer->isAlive())

i didnt see that sorry